src/libostree/ostree-repo-pull.c

```diff
@@ -2772,11 +2834,38 @@ _ostree_repo_load_cache_summary_if_same_sig (OstreeRepo        *self,
   return TRUE;
 }
 
+static void
+store_file_cache_properties (int         dir_fd,
+                             const char *filename,
+                             const char *etag,
+                             guint64     last_modified)
+{
+  glnx_autofd int fd = -1;
+  struct timeval time_vals[] =
+    {
+      { .tv_sec = last_modified, .tv_usec = 0 },  /* access */
```

Comment:
Why reset the access time versus `UTIME_OMIT`?

Reply:
Because I didn’t know about `UTIME_OMIT`. Fixed, thanks